### DISCUSSION TITLE
Release list page using search breaks tables in release notes

### DISCUSSION THREAD

**Discussion by BourgoisMickael** [community/community#126747] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

>  [!IMPORTANT]
> Release notes can contain tables: https://github.com/actions/runner-images/releases

![image](https://github.com/community/community/assets/35171168/06f5b337-ef37-40c6-a112-f58ed6a5f742)

> [!CAUTION]
> But if you use the search feature, tables are broken and displayed as text in the release note: https://github.com/actions/runner-images/releases?q=macOS&expanded=true

![image](https://github.com/community/community/assets/35171168/d9dce455-e8dd-4cab-b95e-ee00032b0926)

**Comment by Emsals1z** [comment]
🫡
